src/styles/container.scss

```diff
@@ -0,0 +1,25 @@
+.container {
+  display: grid;
+  grid-template-columns: 1fr;
+
+  &__block {
+    height: 300px;
+    background-color: rgb(255, 0, 0);
+    text-align: center;
+    line-height: 300px;
+
+    @each $block, $color in $colors {
+      &--#{$block} {
```

Comment:
better use construction with `:nth-child`
that will allow you to get rid of all the number-modified classes (`container__block--6`)

Reply:
Okay great. Thank you.